### DISCUSSION TITLE
Brilliant! 🎉

### DISCUSSION THREAD

**Discussion by exstntlmsnthrp** [Kunzisoft/KeePassDX#1603] [General]
I just wanted to say that the new beta release is brilliant! The new UI and other updates are lovely. It's already one of my most used apps, now it makes want to open it just because it's so well designed. Cheers.

**Comment by J-Jamet** [comment]
Thanks, there are still some corrections to be made but it's nice to see that you like it. Don't hesitate to let me know where I can make improvements.

**Comment by nicoursi** [comment]
Graphically I liked more the previous one

**Reply by J-Jamet** [reply]
In this case, the solutions now are to propose new themes, as the old ones were difficult to maintain.
